### ISSUE TITLE
Exported ONNX policy is fixed at batch size 1 on torch < 2.8

### ISSUE THREAD

**Issue by rakhimovv** [telekinesis-ai/telekinesis-rlbotics#5]
### Summary

`OnPolicyRunner.export()` asks for a dynamic batch axis but traces the model with a batch of one, so on torch < 2.8 the exported graph is **fixed at a batch of one**. There is no warning and no export-time error; the failure appears only at deployment.

`pyproject.toml` requires `torch>=2.6.0`, so this affects the low half of the supported range.

### Where

`src/telekinesis/rlbotics/runner.py`, `export_policy_to_onnx()`:

```python
torch.onnx.export(
    exported,
    (self._export_dummy_input(policy),),   # <- batch of 1
    ...
    dynamic_shapes=({0: torch.export.Dim("batch")},),
)
```

and `_export_dummy_input()` returns `torch.randn(1, policy.input_dim)`.

### Cause

`torch.export.Dim("batch")` allocates the range `[2, inf]` — the 0/1 specialization. A size-one example dimension therefore cannot be made dynamic, and torch specializes it to the literal `1` instead of honoring `dynamic_shapes`. See the [0/1 specialization note in the torch.export docs](https://docs.pytorch.org/docs/main/user_guide/torch_compiler/export.html).

### Reproduction

Exporting the same model with an example batch of 1 vs 2, all else identical:

| torch | example batch 1 | example batch 2 |
|---|---|---|
| 2.6.0 | `[1, 8]` | `['s0', 8]` |
| 2.7.0 | `[1, 8]` | `['batch', 8]` |
| 2.13.0 | dynamic | dynamic |

```python
import torch, torch.nn as nn, onnx, tempfile, os

class Tiny(nn.Module):
    def __init__(self): super().__init__(); self.net = nn.Linear(8, 4)
    def forward(self, x): return self.net(x)

def export(batch):
    path = os.path.join(tempfile.mkdtemp(), "p.onnx")
    torch.onnx.export(
        Tiny().eval(), (torch.randn(batch, 8),), path,
        input_names=["observation"], output_names=["action"],
        opset_version=18, dynamo=True, external_data=False,
        dynamic_shapes=({0: torch.export.Dim("batch")},),
    )
    d = onnx.load(path).graph.input[0].type.tensor_type.shape.dim
    return [(x.dim_param or x.dim_value) for x in d]

print(export(1))  # torch 2.7.0 -> [1, 8]
print(export(2))  # torch 2.7.0 -> ['batch', 8]
```

### Symptom

A vectorized deployment feeds one observation per environment and fails on its first step:

```
onnxruntime InvalidArgument: Got invalid dimensions for input: observation
  index: 0 Got: 64 Expected: 1
```

Reproduced against Isaac Lab v2.3.2 / Isaac Sim 5.1.0 on an RTX PRO 6000 with torch 2.7.0: all 31 `configs/isaaclab/*.yaml` train and export, then fail at the deployment step in `examples/training_example.py`. With the fix below, 29 of 31 pass (the remaining two are unrelated configuration problems, filed separately).

### Why CI does not catch it

`test_matches_the_trained_policy` does use a batch of 5 and would fail — but only on a torch that reproduces the specialization. On a newer torch the axis is dynamic anyway, so the suite is green. `test_batch_dimension_is_dynamic` passes on every version because it only feeds a single *unbatched* observation, which is exactly the case a specialized axis still serves.

### Suggested fix

Trace with a batch of two; the tensor is only a tracing example, so nothing else changes.

```diff
-    def _export_dummy_input(self, policy: MLPModel | CNNModel) -> torch.Tensor:
+    EXPORT_BATCH = 2
+
+    def _export_dummy_input(self, policy: MLPModel | CNNModel) -> torch.Tensor:
         if isinstance(policy, CNNModel):
-            return torch.randn(1, policy.input_channels, *policy.input_dim)
-        return torch.randn(1, policy.input_dim)
+            return torch.randn(self.EXPORT_BATCH, policy.input_channels, *policy.input_dim)
+        return torch.randn(self.EXPORT_BATCH, policy.input_dim)
```

For the regression test, asserting the **graph** rather than running it is what makes it version-independent — a behavioural test passes on whichever torch does not reproduce the bug:

```python
batch_dim = onnx.load(str(path)).graph.input[0].type.tensor_type.shape.dim[0]
assert batch_dim.dim_param   # symbolic; match truthiness, not the name ('s0' on 2.6, 'batch' on 2.13)
```

I have this change plus tests ready as a branch and am happy to open a PR if that is welcome.
